### ISSUE TITLE
Linear Team data object

### ISSUE THREAD

**Issue by meyerkev** [terraform-community-providers/terraform-provider-linear#12]
So I'd like to add some more data objects to support a workflow where we look up a team by name and then get the id (for putting into a NewRelic webhook).  This way, we can have an more human-readable mapping

Unless there's any objections?  I'm not sure what the process for contributing is, but at that point I'll hardcode the ID in our Terraform.

**Comment by pksunkara** [comment]
Yeah, feel free to contribute. I am still working on the resources first.
